config/routes.rb

```diff
@@ -1,23 +1,5 @@
 Katuma::Application.routes.draw do
 
-  mount PublicPages::Engine, at: '/'
-
-  namespace :api do
-    namespace :v1 do
-      post :sessions, controller: :sessions, action: :create
-      delete :sessions, controller: :sessions, action: :destroy
-      get :account, controller: :users, action: :account
-      resources :memberships
-      resources :users, except: [:new, :edit]
-      resources :groups, except: [:new, :edit] do
-        resources :users_units, except: [:new, :edit], shallow: true do
-        end
-      end
-      resources :invitations, except: [:new, :edit] do
-        member do
-          put :accept
-        end
-      end
-    end
-  end
+  mount Account::Engine,    at: '/'
+  mount Landing::Engine,    at: '/'
```

Comment:
Shared only provides mixins and classes to be used in other engines, no routes here.

Ditto for the Landing engine